pkgs/development/tools/database/pgweb/default.nix

```
@@ -0,0 +1,15 @@
+{ buildGoPackage, fetchFromGitHub, ... }:
```

```suggestion
{ buildGoPackage, fetchFromGitHub }:
```

You don't need ellipsis here.

---

👍